messages/run-common.js

```diff
@@ -0,0 +1,23 @@
+module.exports = {
+	"flags": {
+		"formatDescription": "specify results output format",
+		"formatDescriptionLong": "Specifies output format with results written directly to the console.",
+		"normalizesevDescription": "return normalized severity 1 (high), 2 (moderate), and 3 (low), and the engine-specific severity",
+		"normalizesevDescriptionLong": "Returns normalized severity 1 (high), 2 (moderate), and 3 (low), and the engine-specific severity. For the html option, the normalized severity is displayed instead of the engine severity.",
+		"outfileDescription": "write output to a file",
+		"outfileDescriptionLong": "Writes output to a file.",
+		"projectdirDescription": "provide root directory of project",
+		"projectdirDescriptionLong": "Provides the relative or absolute root project directory used to set the context for Graph Engine's analysis. Project directory must be a path, not a glob. Specify multiple values as a comma-separated list.",
+		"sevthresholdDescription": "throw an error when a violation threshold is reached, the --normalize-severity is invoked, and severity levels are reset to the baseline",
```

Comment:
@teresa-allen-sfdc , preference between the version in the file and the following?
```"throw an error when violations of specific or higher severity are detected, and invoke --normalize-severity"```